src/uwtools/tests/drivers/test_forecast.py

```diff
@@ -233,9 +252,9 @@ def test_forecast_run_cmd():
             "core",
             "-depth",
             2,
-            run_cmd="mpiexec",
-            exec_name="test_exec.py",
-        )
+        ]
+        wcoss2_expected = "mpiexec -n 4 -ppn 8 --cpu-bind core -depth 2 test_exec.py"
+        assert wcoss2_expected == fcstobj.run_cmd()
```

Comment:
I'd be ok with that. I think the unit tests are probably a bit far removed from users to be really useful documentation, so maybe actual _documentation_ will be useful at some point. There may well be other flags that need to be used for certain use cases, and different values that need to be used for these invocations depending on circumstances. For example `-n` and `-ppn` values are likely to change based on model resolution and how many nodes are being allocated to a specific job. I don't think there's "one answer" for these things, and misleading advice is IMO worse than remaining silent and saying "Please `man mpiexec` and talk to your HPC admin about how to set this".

I have been thinking, though, that one place `uwtools` really could add value would be in learning, for each driven component, what its memory, threading, etc. requirements really are, and being able to construct, from a fairly simple description of what the user wants to run -- say, FV3 at a given resolution, on a platform with so many cores per node and so much memory per node -- a viable batch resource request and MPI invocation. Think about when we were working on MPAS, and I had to keep tweaking the `ppn` value until it didn't crash. That was just trial-and-error, but it's based on real resource requirements -- I just didn't have time to understand the nuance. But UCAR has published that (what was it?) 0.175 MB of memory needed per grid cell and, although I wasn't able to get a running configuration that seemed harmonious with that, I think that, for each combination of NWP component (of a specific release) and HPC characteristics, there is _some_ formula that calculates working -- and even optimal -- runtime configurations. It would be amazing to be able to calculate for users an optimal configuration if they want to run as fast as possible and don't care about the cost; or to calculate at least a _working_ configuration if they tell us that they only want to use a certain number of nodes. We should even be able to roughly calculate wallclock requirements (of course subject to e.g. unexpected load on the system, so maybe padded -- but this is always a problem). I think these things are all calculable, we just don't know how to calculate them today. But what a boon to NWP users it would be if we could supply those formulas. It'd be tricky, because e.g. GNU builds might run more slowly than MPI builds. But I think the user experience today, trying to figure these things out with very little guidance, is awful. Maybe someday we can improve on it.

I'm fine with whatever you want to do with this test. I don't think the machine names belong here.